libbpf-rs/src/query.rs

```diff
@@ -260,32 +497,84 @@ gen_info_impl!(
 
 /// Information about BPF type format
 #[derive(Debug, Clone)]
-// TODO: Document members.
 #[allow(missing_docs)]
 pub struct BtfInfo {
-    pub btf: u64,
-    pub btf_size: u32,
+    /// The name associated with this btf information in the kernel
+    pub name: String,
```

Comment:
Hm, I thought we agreed on making this `CString`. But now not even the comment shows up anymore...